docs/source/guide/zne-3-options.md

```diff
@@ -216,6 +216,66 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+#### Folding gates by layer
+
+Circuit debugging techniques (as proposed in
+[arXiv:2204.06056](https://arxiv.org/abs/2204.06056)) utilize the ability to
+apply layerwise folding of a circuit.
+
+A quantum circuit $C$ may be represented as a series of $d$ layers. Each layer
+$L$ contains one or more quantum gates acting concurrently on an $n$-qubit
+system:
+
+$$C = L_d L_{d-1} \ldots L_2 L_1$$.
+
+The $i$-inverse operation of a circuit $C$ is defined as
+
+$$C^{(i)} = L_d \ldots L_{i+1} \left[L_i L_i^{-1} L_i\right] L_{i-1} \ldots L_1$$
+
+where we refer to $L_i$ as the *target layer* and $L_i^{-1}$ as the *inverse
+target layer*. The inverse target is composed of gates that would invert the
+gates in the target layer.
+
+One can perform $m$ such inversions on a quantum circuit as well. A multiple
+$i$-inversion is defined as
+
+$$C^{(i)} = L_d \ldots L_{i+1} L_i \left[L_i^{-1} L_i\right]^m L_{i-1} \ldots L_1$$
+
+where we apply a given $i$-inversion $m$ times.
+
+For example, consider the following circuit
+
+```{code-cell} ipython3
+from mitiq.zne.scaling.layer_scaling import layer_folding
```

Comment:
```suggestion
from mitiq.zne.scaling import layer_folding
```

I think we should be able to import directly from `scaling` now.

Reply:
I would think so as well, but attempting to import this seems to yield an error when running on the pipeline for mitiq. I'm keeping the longer import style here, but let me know if you think it's possible to use the shorter one without the pipeline failure.

Reply:
Is there a build where this was tested and it failed? I see https://github.com/unitaryfund/mitiq/pull/1767/commits/a547bc285e14c7b488a2d57b5c7187736d1b1ba2, but when inspecting the build, it seems all the failures I saw was from an unrelated error that was fixed elsewhere.

Reply:
*Think* this should be okay now.